generator/tools/templates/react/README.md

```diff
@@ -0,0 +1,18 @@
+# applet-template
+
+## Dev Instructions
+
+-   clone directly into `[AppletSys]/ui` directory
```

Comment:
Are these instructions for third-party consumption? If so, we may want to specify the `services/user/[Applet]/ui` directory. And will non-system services need to end in "Sys"?

Reply:
Cleared up, made it more generic.